generated/nidcpower/nidcpower/_library.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, ctypes_library):
         # We cache the cfunc object from the ctypes.CDLL object
         self.niDCPower_Abort_cfunc = None
         self.niDCPower_CalSelfCalibrate_cfunc = None
+        self.niDCPower_ClearLatchedOutputCutoffState_cfunc = None
```

Comment:
please update diff as these are unrelated to the PR

Reply:
I had mentioned these in the PR description above:

> Update generated code from earlier, unrelated metadata changes, also in NI-DCPower:
> - Remove attribute IDs from attribute names
> - Add ouput cutoff functions, attributes and enums

These changes were already in the metadata (but not in github), and at some point they'll have to be submitted, or we'll have to keep taking them out manually before each submission. Alex had mentioned in one of our standups that it would be ok to include them with this user story. Do you disagree?

Reply:
I disagree. That PR adds a bunch of new API in the newly/almost released NI-DCPower 20.7 and should be done independently.

I'm going to try to get that done ASAP so it's not blocking you.

Reply:
> That PR adds a bunch of new API in the newly/almost released NI-DCPower 20.7 and should be done independently.

Done here: https://github.com/ni/nimi-python/pull/1580

Reply:
Updated PR after merging in Shreyas' [#1580](https://github.com/ni/nimi-python/pull/1580) changes.